package.json

```diff
@@ -67,8 +67,10 @@
     "@babel/plugin-transform-react-jsx": "^7.7.0",
     "@babel/plugin-transform-runtime": "^7.6.2",
     "@babel/register": "^7.7.0",
-    "@material-ui/core": "^4.6.0",
-    "@material-ui/icons": "^4.5.1",
+    "@emotion/styled": "^11.3.0",
```

Comment:
Why was this dependency added? We don't seem to use it anywhere?

Reply:
I'm also getting errors locally stemming from this dependency due to an inability to resolve `@emotion/react`.

Reply:
Okay. Seems that `@emotion/react` is a peer dependency of `@emotion/styled` and is thus missing from our deps. Still not clear to m why `@emotion/styled` is necessary.

Reply:
Okay. Required by `@mui/material`. Resolved. Added the missing dependency.